modules/ROOT/pages/servers-cert-renewal.adoc

```diff
@@ -9,30 +9,92 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoi
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
-Registering a Mule server in Runtime Manager requires a valid certificate provisioned and signed by Runtime Manager. The certificate ensures secure communications between Runtime Manager and the Runtime Manager agent. 
+Registering a Mule server in Anypoint Runtime Manager requires a valid certificate provisioned
+and signed by Runtime Manager.
+The certificate ensures secure communication between Runtime Manager and the Runtime Manager agent. 
 
-However, certificates expire, and you cannot renew them after they expire. Before a certificate expires, renew it directly from Runtime Manager. You can see the expiration date on the server dashboard.
+When a server certificate expires, Runtime Manager displays `Disconnected` in the *Status* column 
+for the server on the *Servers* page.
+The `mule-agent.log` file displays a message like the following:
 
-A server must be running for you to update its certificate. You can update a certificate for a specific server or multiple servers within a cluster or group.
+----
+com.mulesoft.agent.transport.WSConnection: Failed attempt to connect nro. 1 to the web socket client at mule-manager.anypoint.mulesoft.com: Connection refused.
+----
 
+For applications deployed to the server, Runtime Manager displays `Unknown` in the *Status* column on the *Applications* page.
+
+== Renew a Certificate Before it Expires 
+
+Before a certificate expires, you can renew it directly from Runtime Manager.
+The expiration date appears on the server dashboard:
+
+.The arrow shows the *Certificate expiration date* on the *Settings* page.
```

Comment:
Certificate expiration date can now also be seen in the "Dashboard" page of the dashboard, without having to access the "Settings" page. For nodes of a cluster/group, this is the only way of checking the expiration date of a certificate, as the "Settings" page is not available for them.

Reply:
Thanks - updated screenshot and caption.